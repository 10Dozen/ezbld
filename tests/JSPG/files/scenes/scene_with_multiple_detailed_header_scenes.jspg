# Scene1 | scene_left
*goto: NextScene

# Scene2
*goto: NextScene

# Scene3 | title
*goto: NextScene

# Scene4 | dialog_left pic.jpg
*goto: NextScene

# Scene5 |
*goto: NextScene
