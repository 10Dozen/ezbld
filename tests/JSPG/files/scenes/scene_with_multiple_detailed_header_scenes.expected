[0]
cls = JSPGScene
name = Scene1 | TEXT
type = scene | TEXT
goto = NextScene | TEXT

[1]
cls = JSPGScene
name = Scene2 | TEXT
goto = NextScene | TEXT

[2]
cls = JSPGScene
name = Scene3 | TEXT
type = title | TEXT
goto = NextScene | TEXT

[4]
cls = JSPGScene
name = Scene4 | TEXT
type = dialog_left | TEXT
portrait = pic.jpg | TEXT
goto = NextScene | TEXT

[5]
cls = JSPGScene
name = Scene5 | TEXT
goto = NextScene | TEXT
